lib/items/issueish-detail-item.js

```diff
@@ -150,42 +146,15 @@ export default class IssueishDetailItem extends Component {
     return this.emitter.on('did-change-title', cb);
   }
 
-  terminatePendingState() {
-    if (!this.hasTerminatedPendingState) {
-      this.emitter.emit('did-terminate-pending-state');
-      this.hasTerminatedPendingState = true;
-    }
-  }
-
-  onDidTerminatePendingState(callback) {
-    return this.emitter.on('did-terminate-pending-state', callback);
-  }
-
-  destroy = () => {
-    /* istanbul ignore else */
-    if (!this.isDestroyed) {
-      this.emitter.emit('did-destroy');
-      this.isDestroyed = true;
-    }
-  }
-
-  onDidDestroy(callback) {
-    return this.emitter.on('did-destroy', callback);
-  }
-
   serialize() {
     return {
       uri: this.getURI(),
       deserializer: 'IssueishDetailItem',
     };
   }
 
-  getTitle() {
-    return this.title;
-  }
-
   observeEmbeddedTextEditor(cb) {
```

Comment:
`observeEmbeddedTextEditor` is another "magic hidden Atom item protocol method." It might be worth pulling into `ItemComponent` even if it isn't widely used yet, just to keep it close to the similar methods.

Reply:
sounds good!

Reply:
yea totally!

Reply:
totally!